packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2417,6 +2417,14 @@ export class ContainerRuntime
 			this._orderSequentiallyCalls++;
 			result = callback();
 		} catch (error) {
+			this.mc.logger.sendErrorEvent(
```

Comment:
rather than adding yet another log (there are already 2), can we just fix the error we pass to close such that it doesn't clobber the original error

Reply:
the error passed to close isn't actionable today, as it loses all the original details

Reply:
you could also then add a test that validates the original error is preserved

Reply:
There are no existing logs for this, there are two container close calls which logs as a side effect.

I'm not very happy with the current state, as If we replace the current generic error, we can't differentiate a close event vs a close event coming from `OrderSequentially`, which I think would be valuable. It would also be valuable to know the context of this, including the depth of the order sequentially calls, hence the new log.